api_test.py

```diff
@@ -83,28 +90,37 @@ def generate_simple_query(fields, nodename):
 def main(): 
 	
 	config = get_config('settings_d42.yaml')
-	nodename = 'ubuntu.saltmaster5'
+	if len(sys.argv) < 2:
```

Comment:
Ah nice find: in the api_test file I had left the nodename to a default used in development.